### PR TITLE
fix: itemTypes added via addNewItemType getting sorted to top (#152)

### PR DESCRIPTION
**Pull request by Enduriel**

### CODE REVIEW

File: msu/config/items.nut [hunk 1]
```diff
@@ -38,15 +38,12 @@ foreach (itemType in ::Const.Items.ItemType)
 	}
 
 	local max = 0;
```
Comment (TaroEld) [inline msu/config/items.nut:40]: Leftover variable

**Review by TaroEld — APPROVED** [review]
Remove max then lgtm

File: msu/config/items.nut [hunk 2]
```diff
@@ -37,16 +37,12 @@ foreach (itemType in ::Const.Items.ItemType)
 		return;
 	}
 
-	local max = 0;
-	foreach (w, value in ::Const.Items.ItemType)
-	{
-		if (value > max) max = value;
-	}
+	foreach (itemType, value in ::Const.Items.ItemType) ::Const.Items.ItemType[itemType] = value << 1;
```
Comment (LordMidas) [inline msu/config/items.nut:40]: Won't this cause `None = 0` to become `None = 1`?

Comment (Enduriel) [reply msu/config/items.nut:40]: No, because there are no 1 bits to shift into a 1 (it didn't happen during testing either)

**Review by LordMidas — APPROVED** [review]

**Pull request merged**